### PR TITLE
 gep-2162 - small fixes + bump to experimental

### PR DESCRIPTION
**Pull request by LiorLieberman**
**What type of PR is this?**
/kind cleanup

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by youngnick** [comment]
LGTM, thanks @LiorLieberman.

I'll just approve now though, and wait for further review.

/approve

File: geps/gep-2162.md [hunk 1]
```diff
@@ -177,12 +177,12 @@ Once the GatewayClass features support are is published into the status we could
...
-
 ### Add Gateway API Version field to the GatewayClass Status
 
 We got some feedback that it will be useful to indicate what what Gateway API version the implementation supports. So when we have supported features published in the GatewayClass Status, users will also be able to understand that those are the supported features for a specific Gateway API version.
 
 This work is likely to require its own small GEP but ideally what this field would mean is that an implementation supports Max(vX.X). 
 
 The value of it is to provide a better user experience and also more foundation for tools to be able to warn for example when a GatewayClass and CRDs have mismatched versions.
+
+### Add a table with feature name and description to document what each feature means
```
Comment (robscott) [inline geps/gep-2162.md:188]: Assuming you meant to add some related content below this heading?

**Review by robscott — COMMENTED** [review]
Thanks @LiorLieberman! I think this may be missing a bit of content but otherwise I think it's ready for experimental.

File: geps/gep-2162.md (hunk 1 above)
Comment (robscott) [reply geps/gep-2162.md:188]: Honestly I think it's also fine to just leave it in future work. But to complete this I think we'd want to document how this feature works + what each entry in the list means. So maybe that would live somewhere like https://gateway-api.sigs.k8s.io/api-types/gatewayclass/. With that said, I think a follow up is fine for that. It just feels awfully funny to have a heading with no content below it as the case is now.

Comment (LiorLieberman) [reply geps/gep-2162.md:188]: added a line to describe the work. 
Hopefully this addresses the concern

**Review by shaneutt — APPROVED** [review]
I think it's time to get this moving into experimental and have implementations try it out :+1:

**Comment by shaneutt** [comment]
Uhhh... I think we should have left the unhold for Rob who still had active comments.

**Comment by robscott** [comment]
> Uhhh... I think we should have left the unhold for Rob who still had active comments.

Reviewed this one retroactively and the updates LGTM after all, but yeah, would be good to leave the hold in place in the future.

**Comment by LiorLieberman** [comment]
> Uhhh... I think we should have left the unhold for Rob who still had active comments.

Uhhh... my bad sorry, i was from the Github App and couldn't figure out where the hold came from.

**Pull request merged**